index.rst

```diff
@@ -1,59 +1,374 @@
+----------------------
+
+Calibration production runs intended for broad use (i.e. outputs will be at least candidates for membership in the recommended calibration collection for this instrument) should output to collections with names that start with ``<instrument>/calib/<ticket>/``.
+Those produced for experimental or development purposes should start with ``u/<user>/<ticket>/``.
+
+In either case, the ``RUN`` collections that hold output datasets directly will usually require another disambiguating term, mapping roughly to the expected validity range epoch.
+Actual validity ranges are not assigned until datasets are certified (i.e. added to ``CALIBRATION`` collections), and until then, the usual dataset type + data ID constraint applies (i.e. there can only be one ``bias`` for each detector in a particular ``RUN`` collection).
+
+.. note::
+
+   **TODO**: We should resolve this uncertainty about what the last term should be in calibration production collections by the end of the RFC discussion period.
+   Gen2 used a "CalibDate", which I have always found a bit vague.
+   Some hash of input IDs seems a bit better, but it would need to be computed by external code, because we need the output collection name before we start processing (and also because the user probably wants it in some file before they launch any jobs, so they can easily look up what hashes mean).
+   Either should probably be combined with the timestamp suffixes that ``pipetask`` can automatically add to avoid clashes (especially differences  due to e.g. configuration rather than inputs).
+
+As noted in :ref:`collections-per-instrument`, certified calibration products intended for broad use should go in ``CALIBRATION`` collections named *just* ``<instrument>/calib/<ticket>``.
+``CALIBRATION`` collections can also of course be nested under ``u/<user>/<ticket>``, but may not always be necessary for development work, because a ``RUN`` or ``CHAINED`` collection directly containing e.g. new ``bias`` datasets can also be used as an input to a processing run that generates new ``flat`` datasets (as long as only one calibration epoch is in play).
+
+.. note::
+
+   **TODO**: While the middleware *can* use ``RUN`` collections as inputs to later CPP processing steps, it's up to the CPP team whether they want to permit that, with the alternative being to always certify between steps as a matter of policy.
+   We should resolve this question by the end of the RFC discussion period.
+
+.. note::
+
+   "Curated" calibration datasets that are written from a source-of-truth in an ``obs_*_data`` git repository (rather than generated directly via pipeline processing) are currently written to ``RUN`` collections with names of the form ``<instrument>/calib/curated/<calibDate>``, which are then ingested directly into an ``<instrument>/calib`` ``CALIBRATION`` collection (which clashes with our proposal earlier to make ``<instrument>/calib`` a ``CHAINED`` collection "pointer").
```

Comment:
Using the structure mentioned above matches what the curated calibrations will also do: updated curated calibrations can be ingested to a tree of ticketed collections, split by the same date value that currently exists in the obs_*_data packages.  These calibrations can then be certified into same ticketed location that they would appear had they been generated directly on that ticket.
I don't think we want to call the disambiguating term either the CalibDate or the validStart.  I'm been calling this an estimatedDate while thinking this through, as it's a guess as to what the validStart date will be.  In the case of curated calibrations, this guess is assumed to be perfectly accurate, as the curation process has done the additional validation checks to determine what the validStart will be.